appveyor-win32-x86.yml

```diff
@@ -32,6 +32,7 @@ install:
   # wait-on, npm-run-all and other useful utilities
   # retry installation several times if first fails due to flake
   - npm install || npm install || npm install
+  - npm run set-status -- --state pending --description "on AppVeyor"
```

Comment:
```suggestion
  - npm run set-status -- --state pending --description "on AppVeyor x86"
```

Reply:
is it not supposed to match https://github.com/cypress-io/cypress-test-example-repos/blob/8b79b4229cc36a6d93fcb913f40a74a4a729414a/appveyor-win32-x64.yml#L59-L63